jmx-metrics/src/main/resources/target-systems/jvm.groovy

```diff
@@ -15,8 +15,10 @@
  */
 
 def classLoading = otel.mbean("java.lang:type=ClassLoading")
-otel.instrument(classLoading, "jvm.classes.loaded", "number of loaded classes",
-        "1", "LoadedClassCount", otel.&longValueCallback)
+otel.instrument(classLoading, "process.runtime.jvm.classes.loaded", "Number of classes currently loaded",
+        "{classes}", "LoadedClassCount", otel.&longUpDownCounterCallback)
+otel.instrument(classLoading, "process.runtime.jvm.classes.unloaded", "Number of classes unloaded since JVM start",
+        "{classes}", "UnloadedClassCount", otel.&longUpDownCounterCallback)
```

Comment:
It looks like this should *not* be an UpDownCounter based on the specification, and rather should be a monotonic counter